### ISSUE TITLE
Lots of warnings during compile time 

### ISSUE THREAD

**Issue by gappc** [noi-techpark/it.bz.opendatahub.databrowser#507]
When the Data Browser is build, a list of warnings appear that are related to the way SVG icons are loaded.

## Steps to reproduce

- compile the Data Browser with `npm run build` and take a look at the output

## Explanation

All SVG icons in the Data Browser reside in their own components and can be included in components statically through an import or dynamically using the [IconParser](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/main/databrowser/src/components/utils/IconParser.vue) component.

As result, during compilation a lot of warnings appear stating that when (Icon) components are included statically and dynamically, then those components won't be moved into another chunk.

Example warning:

```
(!) /home/chris/projects/noi/databrowser/it.bz.opendatahub.databrowser/databrowser/src/components/svg/IconDragAndDrop.vue is dynamically imported by /home/chris/projects/noi/databrowser/it.bz.opendatahub.databrowser/databrowser/src/components/utils/IconParser.vue?vue&type=script&setup=true&lang.ts but also statically imported by /home/chris/projects/noi/databrowser/it.bz.opendatahub.databrowser/databrowser/src/domain/cellComponents/components/utils/editList/table/EditListTable.vue?vue&type=script&setup=true&lang.ts, dynamic import will not move module into another chunk.
```

The question is, what to do with this warnings.

The `IconParser` component seems to be used in `QuickView` components only, and once in the [GpsPointMap](https://github.com/noi-techpark/it.bz.opendatahub.databrowser/blob/main/databrowser/src/components/map/GpsPointMap.vue#L4), although there a static import would suffice.

@sseppi @MatteoBiasi @a-crea what are your opinions?

**Comment by gappc** [comment]
@sseppi as far as I remember, the dynamic icon import was implemented as part of the quick view, but the quick view was removed some time ago.

At the moment, there are three code parts in the entire code base that use dynamic icons, with one single place that doesn't hard-code the icon name but determines it in a "if....else" block. That block could easily be modified to work without dynamic icons.

Therefor, in my opinion, there is no need to have the icons dynamically imported, static imports should suffice.

**Comment by sseppi** [comment]
@mrabans what is your opinion here?

**Comment by mrabans** [comment]
@sseppi @gappc that's a nice to have while doing some refactoring. If there is no advantage in importing it dynamically (as it seems for now) then import them statically. Though other priorities first.

**Comment by sseppi** [comment]
@mrabans I agree, but it would be nice to solve and close the issue in the next 2 iterations.

**Comment by sseppi** [comment]
@gappc super, I close the issue.
